src/core/dfn-panel.js

```diff
@@ -49,7 +49,7 @@ function createPanel(dfn) {
         <a class="self-link" href="${href}">Permalink</a>
         ${dfnExportedMarker(dfn)}
       </div>
-      <h2>Referenced in:</h2>
+      <b>Referenced in:</b>
```

Comment:
Should it be marked as an aria heading? It kinda serves as a landmark.

Reply:
Ideally yes, but I think it's a matter of time pubrules checker starts treating them same as h2.
Note to self: `<b role="heading" aria-level="2">Referenced in:</b>`